src/Shared/Recruit.Vacancies.Client/Infrastructure/Reports/CsvBuilder.cs

```diff
@@ -43,49 +48,42 @@ private static CsvWriter GetCsvWriter(TextWriter streamWriter)
             return csv;
         }
 
-        private void WriteNoResults(CsvWriter csv, DateTime reportDate)
+        private void WriteNoResults(CsvWriter csv, IEnumerable<KeyValuePair<string, string>> headers)
         {
-            WriteTotalHeader(csv, reportDate, 0, 2);
+            WriteTotalHeader(csv, headers);
         }
 
-        private void WriteTotalHeader(CsvWriter csv, DateTime reportDate, int total, int fieldCount)
+        private void WriteTotalHeader(
+            CsvWriter csv,
+            IEnumerable<KeyValuePair<string, string>> headers)
         {
+            headers = headers ?? new KeyValuePair<string, string>[0];
             csv.WriteField("PROTECT");
 
-            for (var i = 1; i < fieldCount; i++)
-            {
-                csv.WriteField("");
-            }
-            csv.NextRecord();
-
-            WriteEmptyRow(csv, fieldCount);
-            WriteEmptyRow(csv, fieldCount);
-
-            csv.WriteField("Date");
-            csv.WriteField("Total_Number_Of_Applications");
+            WriteEmptyRow(csv, 0);
```

Comment:
Guess you didn't see my Slack reply.
```
The reason I didn't call WriteEmptyRow is that it isn't an empty row.  Its starts from i = 1 because the first field says "PROTECT"
```

Reply:
WriteEmptyRow(csv, 0) will write zero commas and then move to the next line. You don't need a comma per empty cell on these rows.

Reply:
The brief for the Provider report was to make it as close to the existing v1 report as possible. This is changing the format.  We can keep this change (I agree having extra commas is not required) and fix if anyone complains. :+1:

In that case I think you'll need to remove one the the `WriteEmptyRow(csv, 0);`  calls I think its only supposed to be 2 empty lines.

Reply:
```
            csv.WriteField("PROTECT");

            WriteEmptyRow(csv, 0);
            WriteEmptyRow(csv, 0);
            WriteEmptyRow(csv, 0);
```

That is 1 line ending for PROTEXT, then 2 empty lines :)